hack/dind-cluster.sh

```diff
@@ -221,7 +221,7 @@ function wait-for-cluster() {
   oc="$(os::build::find-binary oc)"
 
   # wait for healthz to report ok before trying to get nodes
-  os::util::wait-for-condition "ok" "${oc} get --config=\"${kubeconfig}\" --raw=/healthz" "120"
```

Comment:
``provision_util.sh`` isn't used here.  That's legacy and only used by vagrant.  I'm assuming you meant ``images/dind/node/openshift-dind-lib.sh``, which needs to be a separate file so it can be distributed in the dind image.  I'm happy to have ``os::cmd`` take over responsibility for this use case, but it would have to be copied into the image file for distribution regardless.